### DISCUSSION TITLE
The batch of read-in data affects the model's effectiveness

### DISCUSSION THREAD

**Discussion by bupt-yiwo** [pratyushasharma/laser#24] [General]
In order to increase the speed of running the dataset, I have changed the single batch of the original code to 64 and 128 batches, and found that there is a significant drop in the effectiveness of the modified model, is there any solution for this?

**Comment by dkmisra** [comment]
@bupt-yiwo can you provide more details on which model you tried? For GPTJ and LLama2 we ran evaluation with a batch size of 1 and, unfortunately, the code currently doesn't support larger batch sizes. For Roberta, we did use a bigger batch size cause it is a much smaller model.

**Reply by bupt-yiwo** [reply]
I tried it on vicuna-7b-v1.5, which is a fine-tuned version of llama. If it works as you say, then indeed the results should be worse. Thanks!
